app/jobs/cache/feed_job.rb

```diff
@@ -19,7 +19,11 @@ def perform(url, download_media = false)
         end
       end
 
-      Rails.cache.write("feed/#{url}", @feed)
+      cached_feed = Rails.cache.fetch("feed/#{url}")
+      if cached_feed.blank? || cached_feed.last_modified != @feed.last_modified
```

Comment:
On reflection, I think we should let `Feedjira` do the work for us here in detecting whether the feed has been updated, i.e. something like:
```ruby
# @see https://github.com/feedjira/feedjira/blob/v2.1.0/lib/feedjira/feed_utilities.rb#L53
cached_feed.update_from_feed(@feed)
if cached_feed.updated?
  # ...
end
```